### ISSUE TITLE
about the training set

### ISSUE THREAD

**Issue by lyq216398** [Gatedip/GDIP-Yolo#10]
Did you split the training set into the training set and the validation set to select the best epoch or did you just use all the training sets to select the last epoch for testing?
